File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/AttachmentsMessages.scala
package org.openapitools.server.model


/**
 * @param dateSent Time when the message was sent as a UNIX timestamp multiplied by 1000 (matching the format of getTime() in JavaScript).  **Changes**: Changed in Zulip 2.2 (feature level 22).  This field was previously strangely called `name` and was a floating point number.  for example: ''null''
 * @param id The unique message ID.  Messages should always be displayed sorted by ID.  for example: ''null''
*/
final case class AttachmentsMessages (
  dateSent: Option[Int],
  id: Option[Int]
)

File: kotlin-spring/src/main/kotlin/org/openapitools/model/AttachmentsMessages.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param dateSent Time when the message was sent as a UNIX timestamp multiplied by 1000 (matching the format of getTime() in JavaScript).  **Changes**: Changed in Zulip 2.2 (feature level 22).  This field was previously strangely called `name` and was a floating point number. 
 * @param id The unique message ID.  Messages should always be displayed sorted by ID. 
 */
data class AttachmentsMessages(

    @field:JsonProperty("date_sent") val dateSent: kotlin.Int? = null,

    @field:JsonProperty("id") val id: kotlin.Int? = null
) {

}

